tests/functional/idempotency/conftest.py

```diff
@@ -246,3 +253,58 @@ def _func_echo_decoder(self, value):
 @pytest.fixture
 def mock_function():
     return mock.MagicMock()
+
+
+@pytest.fixture
+def static_pk_value():
+    return "static-value"
+
+
+@pytest.fixture
+def expected_params_update_item_compound_key_static_pk_value(
```

Comment:
@heitorlessa, I made a slight update to reduce fixture copy/pasta: https://github.com/Tankanow/aws-lambda-powertools-python/pull/1

Reply:
If you want, I can submit an actual PR now or you can just update this one.

Reply:
@heitorlessa, thanks for migrating this over. I should have just submitted a PR in this repo. I didn't do that originally b/c I wanted to show the regression without cluttering the PR list here.

Reply:
I did that the minute you pushed to your fork ;) I'm gonna merge this now so I can go to sleep.

Reply:
no worries at all @Tankanow. I'm glad you took the time to create test because that would've slowed us down by an hour otherwise. It was pretty efficient to test the regression - in fact, we never had tests for that specific feature